airflow/ui/src/queries/useClearTaskInstances.ts

```diff
@@ -59,18 +59,25 @@ export const useClearTaskInstances = ({
     if (variables.requestBody.dry_run) {
       onSuccessDryRun(data);
     } else {
-      const taskInstanceKeys = (variables.requestBody.task_ids ?? [])
-        .map((taskId) => {
-          const runId = variables.requestBody.dag_run_id;
+      const taskInstanceKeys = [
+        ...new Set(
```

Comment:
Maybe a comment here on why we can encounter multiple times the same task_id. (people clearing multiple map index of the same task_id).